massa-client/src/cmds.rs

```diff
@@ -294,6 +297,12 @@ impl Command {
                 Ok(Box::new(()))
             }
 
+            // TODO: process spawn should be detached
```

Comment:
Is there an issue for that ?

Reply:
Yes! We got a discussion about his here: https://github.com/massalabs/massa/issues/1609:

> - @damip should be started detached so that when we close the client, it doesn't close the node
> - @yvan-sraka I guess the right way of doing it is https://stackoverflow.com/a/53478775/2879157
> - @damip needs to be portable
> - @yvan-sraka grrr ... that's true, hmm ... `Command::detach` was removed from `std::process` API rust-lang/rust#17176 so I guess something like https://docs.rs/fork/ will do the job :)
> - @damip yeah we can try. Ideally, we need a mechanism to avoid starting 2 instances by mistake
> - @yvan-sraka Euhm like a mutex? We could check out if there is a process massa-node already running, an implementation could be massa-node that creates an empty lock file that the client could detect? 
> - @damip Or a health check API endpoint, I'm not sure...
> - @yvan-sraka So we authorize, on the same machine, running two nodes with different ports? or we try to forbid that? Maybe it's a good (soft) security protection to say that one machine == one node (unless we deploy several parallel Massa-based networks)
> - @yvan-sraka Do we postpone this study/discussion in a follow-up issue?
> - @damip yes, maybe remove this command for now from the client so that we can merge and do a clean follow-up to add it
> - @damip yeah let's do it in a followup

And are the follow-up issue -> https://github.com/massalabs/massa/issues/1214 and closed PR https://github.com/massalabs/massa/pull/1741 :smiling_face_with_tear:

Reply:
So that won't be done ? :confused:

Reply:
I guess something like https://docs.rs/fork/ will still do the job :slightly_smiling_face:! I should update the PR.